text/exercise2/world_cup.md

```diff
@@ -1,12 +1,10 @@
 # Women's world cup
 
-The Women's World Cup stands as the pinnacle of global competition 
-in the NFL, showcasing the finest female athletes in intense matchups 
-on the gridiron. This prestigious tournament, organized by FIFA, 
-captivates audiences worldwide with its electrifying display of 
-athleticism, strategic plays, and nail-biting touchdowns. Teams from 
-across the globe, armed with skillful quarterbacks and powerful 
-offensive lines, battle for supremacy in this exhilarating event.
+The Women's World Cup is a spectacular showcase of athleticism, skill, 
+and passion, bringing together the finest female soccer players from 
+around the globe. This prestigious tournament, organized by FIFA, 
+serves as a platform to celebrate women's football, fostering inclusivity
```

Comment:
```suggestion
serves as a platform to celebrate women's soccer, fostering inclusivity
```
Suggest using soccer throughout for consistency. Perhaps we could include a note at some point mentioning that the sport is called football in Europe.

Alternatively, does FIFA use the term "football"? If so, suggest we use that term throughout to be consistent with the terms used by the Women's World Cup. In this case, a small note to say that the sport is known as "soccer" some forms of English should suffice.

Alternatively, does FIFA use the term "football"? If so, suggest we use that term throughout to be consistent with the terms used by the Women's World Cup. In this case, a small note to say that the sport is known as "soccer" some forms of English should suffice.